src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -104,22 +137,23 @@ static void Main(string[] args)
                             process.StartInfo.UseShellExecute = false;
                             process.StartInfo.RedirectStandardOutput = true;
                             process.StartInfo.FileName = "Akka.NodeTestRunner.exe";
-                            process.StartInfo.Arguments = String.Format(@"-Dmultinode.test-assembly=""{0}"" -Dmultinode.test-class=""{1}"" -Dmultinode.test-method=""{2}"" -Dmultinode.max-nodes={3} -Dmultinode.server-host=""{4}"" -Dmultinode.host=""{5}"" -Dmultinode.index={6}",
-                                assemblyName, nodeTest.TypeName, nodeTest.MethodName, test.Value.Count, "localhost", "localhost", nodeTest.Node - 1);
+                            process.StartInfo.Arguments =
+                                $@"-Dmultinode.test-assembly=""{assemblyName}"" -Dmultinode.test-class=""{
+                                    nodeTest.TypeName}"" -Dmultinode.test-method=""{nodeTest.MethodName
+                                    }"" -Dmultinode.max-nodes={test.Value.Count} -Dmultinode.server-host=""{"localhost"
+                                    }"" -Dmultinode.host=""{"localhost"}"" -Dmultinode.index={nodeTest.Node - 1
+                                    } -Dmultinode.listen-address={listenAddress} -Dmultinode.listen-port={listenPort}";
                             var nodeIndex = nodeTest.Node;
-                            process.OutputDataReceived +=
-                                (sender, line) =>
-                                {
-                                    //ignore any trailing whitespace
-                                    if (string.IsNullOrEmpty(line.Data) || string.IsNullOrWhiteSpace(line.Data)) return;
-                                    string message = line.Data;
-                                    if (!message.StartsWith("[NODE", true, CultureInfo.InvariantCulture))
-                                    {
-                                        message = "[NODE" + nodeIndex + "]" + message;
-                                    }
-                                    PublishToAllSinks(message);
-                                };
-
+                            //TODO: might need to do some validation here to avoid the 260 character max path error on Windows
+                            var folder = Directory.CreateDirectory(Path.Combine(OutputDirectory, nodeTest.MethodName));
```

Comment:
Sets up STD OUT logging to capture the full trace for all nodes into their own files, all grouped together into a folder that shares the name of the test. Had some issues where I hit the 260 character path limit earlier, so we might need to possibly hash or shorten the name of the folder.